packages/components/molecules/f-searchbox/src/services/general.services.js

```diff
@@ -103,9 +103,22 @@ const fullAddressLocalStorageService = {
     }
 };
 
+/**
+ * Extract postcode from address in agreed format
+ * @param {string} address A address string
+ */
+const extractPostcode = address => {
+    try {
+        return address.split(',')[0];
+    } catch (e) {
+        return null;
```

Comment:
Maybe worth adding some logging in future.

Reply:
This part of the method shouldn't ever be hit, unless an address isn't passed in. We could get this to `throw` instead in those cases?